Casks/xampp-vm.rb

```diff
@@ -2,7 +2,7 @@
   version "8.1.6-0"
   sha256 "11be67f9c8bd955a07bbc93af9cf126fda20ba2b60eef66f8c92bb655776cedb"
 
-  url "https://www.apachefriends.org/xampp-files/#{version.split("-").first}/xampp-osx-#{version}-vm.dmg"
+  url "https://downloadsapachefriends.global.ssl.fastly.net/#{version.split("-").first}/xampp-osx-#{version}-vm.dmg"
```

Comment:
```suggestion
  url "https://downloadsapachefriends.global.ssl.fastly.net/#{version.split("-").first}/xampp-osx-#{version}-vm.dmg",
    verified: "downloadsapachefriends.global.ssl.fastly.net"
```
You need to add fastly as a verified domain here (see https://docs.brew.sh/Cask-Cookbook#when-url-and-homepage-domains-differ-add-verified) :)